README.md

```diff
@@ -123,25 +123,29 @@ await initialize({
 Additionally, this library exposes 23 modules that include the vscode version of some services (with some glue to make it work with monaco):
 
 - **Extensions** (included by default): `vscode/service-override/extensions`
-  - Support for VSCode extensions.
+  - Support for VSCode extensions. A worker configuration can be past to it:
+    - Then, everything runs in one worker, where extensions run in an iframe, with all the implications (can be created by the bundler directly). The worker script is expected to be hosted on a separate domain.
 - **Files** (included by default): `vscode/service-override/files`
-  - It adds the memory filesystem for `file://` files, but also adds the support for lazy loaded extension files, separate memory user files (e.g. config, keybindings), cache, logs, etc.
+  - It adds the memory filesystem for `file://` files, but also adds the support for lazy loaded extension files. It adds separate memory user files (e.g. config, keybindings), cache files and log files.
 - **QuickAccess** (included by default): `vscode/service-override/quickaccess`
-  - Enables the quickaccess menu in the editor
+  - Enables the quickaccess menu in the editor (press F1)
```

Comment:
or ctrl+shift+p/ctrl+p like in vscode :)

Reply:
<F1> works in vscode 😉

Reply:
see commit ⬇️